### ISSUE TITLE
Does not work with node v6

### ISSUE THREAD

**Issue by djMax** [aerospike/aerospike-client-nodejs#121]
This is the only module of many that we use that does not work with node v6.

```
CHECK
   [✓] /myproject/node_modules/aerospike/aerospike-client-c/package/usr/local/lib/libaerospike.a
   [✓] /myproject/node_modules/aerospike/aerospike-client-c/package/usr/local/include/aerospike/aerospike.h
   [✓] /myproject/node_modules/aerospike/aerospike-client-c/package/usr/local/aerospike/client/sys/udf/lua/aerospike.lua

  TOUCH Release/obj.target/aerospike-client-c.stamp
  CXX(target) Release/obj.target/aerospike/src/main/aerospike.o
In file included from ../src/main/aerospike.cc:25:
In file included from ../src/include/client.h:20:
../../nan/nan.h:590:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback
      ~~~~~~~~~~~~~^
../../nan/nan.h:596:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback) {
      ~~~~~~~~~~~~~^
../../nan/nan.h:601:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback
      ~~~~~~~~~~~~~^
../../nan/nan.h:607:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback) {
      ~~~~~~~~~~~~~^
4 errors generated.
make: *** [Release/obj.target/aerospike/src/main/aerospike.o] Error 1
```

**Comment by djMax** [comment]
May be that you need to bump the nan dependency.

**Comment by djMax** [comment]
#122

**Comment by jhecking** [comment]
Fixed by #122. Thanks for the PR, @djMax!
